### DISCUSSION TITLE
Calendar versioning

### DISCUSSION THREAD

**Discussion by sheldonhull** [miniscruff/changie#625] [Q&A] [answered]
@miniscruff  How did you configure changie to not using semver numbers as the main tracking and to use Calver style format? If i misunderstood your prior posts then just mark this as closed if it doesn't support that. Thanks!

**Comment by miniscruff** [comment]
For our APIs I used the latest API route prefix and the date of year, month, day and hour. For us this is enough granularity without being too verbose. 

`changie batch v1.$(date -u %Y%m.%d%H)`.
The first 1 correlates to `/api/v1/`. This does limit is to only being able to release once in an hour but at our current speed that's not an issue. 

The semver sort rules will work as normal as long as you go in least to most frequently changing timings.

**Reply by miniscruff** [reply, marked as answer]
Going to add one more comment to this to say that semver doesn't support zero prefix values so the actual date string will look like this:
`changie batch v1.$(date -u +%Y%m.%-d%H)`

The dash before d says to remove the zero prefix, you will get a semver error for the first 9 days of a month if you don't do this.
